docs/docs/native-docker.md

```diff
@@ -304,11 +302,10 @@ container type "MESOS" and a "docker" object.
 }
 ```
 The Mesos containerizer does not support the same parameter options as the Docker containerizer yet.
-The only properties recognized by both containerizers are "image" and "forcePullImage",
+The only properties recognized by both containerizers are "image" and "forcePullImage,"
```

Comment:
I tend to put commas within quotation marks, but I can see the argument that it might make a user think the property is `forcePullImage,`